README.md

```diff
@@ -1 +1,32 @@
-# hw01
\ No newline at end of file
+## Zachary Weinroth
+## October 7, 2019
+## HW01
+
+### The Beginning
+Zach Weinroth was born May 8, 1998 in Summit County, New Jersey.  His parents feared the effects of their child growing up in Jersey and sought out career options in other places. They were able to find work in Denver, Colorado, but they had to weigh the options against each other to be sure it was the right place for the Weinroth family.
+
+New Jersey | Colorado
+---------- | --------
+Close to New York City | The center of the country
+Close to the Atlantic Ocean | Last of NA lands to be submerged by rising sea levels
+... | Nature is everywhere
+... | Land is cheap and spacious, room for kids to play
+It's *New Jersey* | It's **Colorado**
+
+The list made it clear that Denver was the best place for the family and that New Jersey didn't have enough positives to stay:
+
+![Image of Heimlich](Heimlich.JPG)
```

Comment:
Both images didn't load correctly when I reviewed the assignment. Review the markdown for inserting images.